scale-ui/app/modules/jobs/partials/jobDetailTemplate.html

```diff
@@ -27,11 +27,20 @@ <h4 class="panel-title error-title"><strong>{{ vm.job.error.category }} Error: {
                         {{ vm.job.error.description }}
                     </div>
                 </div>
+                <div class="panel panel-info" ng-show="vm.job.is_superseded">
+                    <div class="panel-heading">
+                        <div class="panel-title">
+                            <strong>SUPERSEDED {{ vm.supersededChange }}</strong>
+                        </div>
+                    </div>
+                </div>
```

Comment:
What do you think? 
![image](https://cloud.githubusercontent.com/assets/15909313/18964324/5bab3da2-8646-11e6-8132-c2ee16550c7b.png)